src/include/rbd/librbd.h

```diff
@@ -1499,6 +1499,8 @@ CEPH_RBD_API int rbd_group_list2(rados_ioctx_t p, rbd_group_spec_t *groups,
                                  size_t *size);
 CEPH_RBD_API int rbd_group_get_id(rados_ioctx_t p, const char *group_name,
                                   char *group_id, size_t *size);
+CEPH_RBD_API int rbd_group_get_name(rados_ioctx_t p, const char *group_id,
```

Comment:
Shouldn't there be a C++ version of this API?

Reply:
I didn't add a C++ version since there are no consumer for it, e.g. a CLI that calls it

Reply:
Let's add one for consistency.  For someone working with `IoCtx` instead of `rados_ioctx_t`, running into some minor API missing like this will be a bummer.